### ISSUE TITLE
Floating "+" button to add new source

### ISSUE THREAD

**Issue by dgruano** [manulera/ShareYourCloning_frontend#194]
When the history is long enough, the button to add a new source stays at the bottom of the "tree", so scrolling up to see the beginning of the history hides the "+" button.

What about fixing it with respect to the screen, so that it is always floating and visible? Like chatbots in many websites.

**Comment by manulera** [comment]
Yes, I had thought about something in those lines, perhaps something like this showing at the bottom-right?

https://v4.mui.com/components/floating-action-button/
